gokart/build.py

```diff
@@ -9,16 +9,18 @@
 from gokart.utils import check_config, read_environ
 
 
-class HideLogger:
-    def __init__(self, verbose: bool):
-        self.verbose = verbose
+class LoggerConfig:
+    def __init__(self, level: Optional[int]):
         self.logger = getLogger(__name__)
         self.default_level = self.logger.level
+        self.level = level
 
     def __enter__(self):
-        if not self.verbose:
+        if self.level is not None:
+            logging.disable(self.level)
+            self.logger.setLevel(self.level)
+        if self.level == logging.CRITICAL:
             logging.disable(sys.maxsize)
```

Comment:
Why disable if set to critical?

Wouldn't the following be better?
```suggestion
        if self.level is None:
            logging.disable(sys.maxsize)
```

Reply:
Thanks! Your suggestion is right! Fixed!